.github/workflows/build_addon.yml

```diff
@@ -0,0 +1,271 @@
+
+      - name: Download Godot
+        run: |
+          curl -L -O https://github.com/godotengine/godot/releases/download/${{ env.godot-version }}/Godot_v${{ env.godot-version }}_win64.exe.zip
+          unzip -o Godot_v${{ env.godot-version }}_win64.exe.zip -d godot
+
+      - name: Setup
+        shell: bash
+        run: |
+          python ./setup.py --godot-binary ./godot/Godot_v${{ env.godot-version }}_win64.exe --custom-opencv-dir C:/tools/opencv/build
+
+      - name: Build
+        env:
+          BAZEL_VC: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC"
+        shell: bash
+        run: |
+          source ./venv/Scripts/activate
+          python ./build.py desktop --type release
+
+      - name: Upload build files (artifacts)
+        uses: actions/upload-artifact@v4
+        with:
+          name: build-artifact-windows
+          path: |
+            mediapipe/bazel-bin/GDMP/desktop/*.dll
+
+
+  # Build the Linux Artifacts
+  build-linux:
+    name: 🐧 Build Linux
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.9"
+
+      - name: Setup Bazelisk
+        uses: bazelbuild/setup-bazelisk@v3
+
+      - name: Prerequisites
+        run: |
+          sudo apt-get install -y \
+            ffmpeg \
+            libegl1-mesa-dev \
+            libopencv-core-dev \
+            libopencv-highgui-dev \
+            libopencv-calib3d-dev \
+            libopencv-features2d-dev \
+            libopencv-imgproc-dev \
+            libopencv-video-dev \
+            libopencv-contrib-dev
+          sed -i 's/#"include\/opencv4/"include\/opencv4/g' mediapipe/third_party/opencv_linux.BUILD
+
+      - name: Download Godot
+        run: |
+          curl -L -O https://github.com/godotengine/godot/releases/download/${{ env.godot-version }}/Godot_v${{ env.godot-version }}_linux.x86_64.zip
+          unzip -o Godot_v${{ env.godot-version }}_linux.x86_64.zip -d godot
+
+      - name: Setup
+        run: |
+          python ./setup.py --godot-binary ./godot/Godot_v${{ env.godot-version }}_linux.x86_64
+
+      - name: Build
+        run: |
+          source ./venv/bin/activate
+          python ./build.py desktop --type release
+
+      - name: Upload build files (artifacts)
+        uses: actions/upload-artifact@v4
+        with:
+          name: build-artifact-linux
+          path: |
+            mediapipe/bazel-bin/GDMP/desktop/*.so
+
+
+  # Build the Android Artifacts
+  build-android:
+    name: 🤖 Build Android
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Free Disk Space
+        uses: jlumbroso/free-disk-space@main
+        with:
+          android: false
```

Comment:
May we pin this action at specific version/tag?

Reply:
This is now locked to v1.3.1 - the latest version published.